link-grammar/prepare/build-disjuncts.c

```diff
@@ -303,7 +305,7 @@ build_disjunct(Sentence sent, Clause * cl, const char * string,
 			ndis->category =
 				malloc(sizeof(ndis->category) * ndis->num_categories_alloced);
 			ndis->num_categories = 1;
-			sscanf(string, "%x", &ndis->category[0].num);
+			sscanf(string, " %x", &ndis->category[0].num);
```

Comment:
Using `ndis->category[0].num = strtol(string, NULL, 16);` here makes the code more readable (to me). 16 means base-16.